planning/obstacle_cruise_planner/config/obstacle_cruise_planner.param.yaml

```
@@ -222,3 +222,5 @@
           sudden_object_acc_threshold: -1.1 # If a stop can be achieved by a deceleration smaller than this value, it is not considered as “sudden stop".
           sudden_object_dist_threshold: 1000.0 # If a stop distance is longer than this value, it is not considered as “sudden stop".
           abandon_to_stop: false # If true, the planner gives up to stop when it cannot avoid to run over while maintaining the deceleration limit.
+      limit_margin_for_unknown: 3.0
+      preferred_acc_for_unknown: -3.0
```

```suggestion
      limit_margin_for_unknown: 6.0
      preferred_acc_for_unknown: -6.0
```
とすると、このPR前後で減速の仕様はそのままになるという理解であっているでしょうか？
もしそうなら、デフォルトパラメータは安全側に振ってそのままにしておきたいです。
autoware_launch側で値を更新していきましょう。https://github.com/tier4/autoware_launch.x2/pull/822#pullrequestreview-2403664120

---

はい。このようにパラメータを設定すると、PR前後で全く同一の挙動をするようになります。
仰る通りですので更新します。